app/uploaders/footer_image_uploader.rb

```diff
@@ -0,0 +1,76 @@
+class FooterImageUploader < CarrierWave::Uploader::Base
+  include CarrierWave::Compatibility::Paperclip
+
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  # include CarrierWave::MiniMagick
```

Comment:
This is TODO-esque.  If we think it's feasible we'll include these, let's ticket it.  Otherwise I'd propose we remove these lines to avoid having general CarrierWave docs in code.

```suggestion
```

Reply:
59fbd54